src/main/java/forestry/plugins/PluginManager.java

```diff
@@ -228,10 +275,6 @@ public static void runPreInit(Side side) {
 			if (ForestryAPI.enabledPlugins.contains(ForestryPluginUids.BUILDCRAFT_STATEMENTS)) {
 				plugin.registerTriggers();
 			}
-			if (ForestryAPI.enabledPlugins.contains(ForestryPluginUids.STORAGE)) {
-				plugin.registerBackpackItems();
-				plugin.registerCrates();
```

Comment:
Those needed to be moved to the stage after registering blocks and items.
Maybe one should introduce new stages for this?

The Registry events are executed in alphabetical order by registry names except that items always run after blocks. 
(see forge api documentation)